dydx-testnet-4/pregenesis.json

```diff
@@ -0,0 +1,1750 @@
+{
+  "genesis_time": "2023-10-17T17:00:00Z",
+  "chain_id": "dydx-testnet-4",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "4194304",
+      "max_gas": "-1"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {
+      "app": "0"
+    }
+  },
+  "app_hash": "",
+  "app_state": {
+    "07-tendermint": null,
+    "assets": {
+      "assets": [
+        {
+          "id": 0,
+          "symbol": "USDC",
+          "denom": "ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5",
+          "denom_exponent": "-6",
+          "has_market": false,
+          "market_id": 0,
+          "atomic_resolution": -6
+        }
+      ]
+    },
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": []
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "dydx1zlefkpe3g0vvm9a4h0jf9000lmqutlh9jwjnsv",
+          "coins": [
+            {
+              "denom": "dv4tnt",
+              "amount": "1000000000000000000000000000"
+            }
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [
+        {
+          "description": "The native token of the network",
+          "denom_units": [
+            {
+              "denom": "dv4tnt"
+            },
+            {
+              "denom": "dv4tnt",
+              "exponent": 18
+            }
+          ],
+          "base": "dv4tnt",
+          "name": "DYDX v4 Public Testnet Coin",
+          "display": "dv4tnt",
+          "symbol": "dv4tnt"
+        }
+      ],
+      "send_enabled": []
+    },
+    "blocktime": {
+      "params": {
+        "durations": [
+          "300s",
+          "1800s"
+        ],
+        "clock_drift_grace_period_duration": "5s"
+      }
+    },
+    "bridge": {
+      "event_params": {
+        "denom": "dv4tnt",
+        "eth_chain_id": 9,
+        "eth_address": "0xcca9D5f0a3c58b6f02BD0985fC7F9420EA24C1f0"
+      },
+      "propose_params": {
+        "max_bridges_per_block": 10,
+        "propose_delay_duration": "60s",
+        "skip_rate_ppm": 800000,
+        "skip_if_block_delayed_by_duration": "5s"
+      },
+      "safety_params": {
+        "is_disabled": false,
+        "delay_blocks": 86400
+      },
+      "acknowledged_event_info": {
+        "next_id": 99,
```

Comment:
@tqin7 can you confirm what `acknowledged_event_info` values we should use for Testnet #4?